cli_tools/daisy/print_utils.go

```diff
@@ -0,0 +1,34 @@
+package main
```

Comment:
Move it to a shared location, maybe cli_tools/common/utils/performance or similar, in case we want to print performance of Daisy based wrappers (import/export/ovf). 

Also, add copyright comment

Reply:
sounds good